api/attestationdataopts.go

```diff
@@ -23,4 +26,6 @@ type AttestationDataOpts struct {
 	Slot phase0.Slot
 	// CommitteeIndex is the committee index for which the data is obtained.
 	CommitteeIndex phase0.CommitteeIndex
+	// Version is the version of the chain we are using. Used to control fork specific logic.
+	Version spec.DataVersion
```

Comment:
Do we need this, given that we are passed the slot and the library knows the epochs of the hard fork boundaries?

Reply:
I wasn't entirely sure how to do this so I've made an attempt to use the Spec to retrieve the electra epoch and convert. Please let me know if there is a better way to achieve this